superbench/benchmarks/micro_benchmarks/cudnn_function.py

```diff
@@ -440,14 +442,23 @@ def _process_raw_result(self, cmd_idx, raw_output):
                     self._curr_run_index, cmd_idx, self._name, raw_output, str(e)
                 )
             )
-            return False
+            error = True
         if error:
-            logger.error(
-                'Error in running cudnn test - round: {}, index of cmd: {}, benchmark: {}, raw data: {}'.format(
-                    self._curr_run_index, cmd_idx, self._name, raw_output
+            if self._capability == 9.0 and 'CUDNN_STATUS_NOT_SUPPORTED' in raw_output:
```

Comment:
is it possible to just disable the configs that cannot run?
some configs also fail on A100, and the error code can also be CUDNN_STATUS_NOT_INITIALIZED

Reply:
disable the configs looks weird, I think for this pr, we just make it be able to run even there's one command failed, then it should work for both cases